proto/Makefile

```diff
@@ -0,0 +1,28 @@
+PB2_OUT_BASE_DIR := pb2
+PB2_OUT_DIR := $(PB2_OUT_BASE_DIR)/otaclient_pb2
+
+.PHONY: all
+all: whl
+
+.PHONY: clean
+clean:
+	@rm -rf build *.egg-info dist $(PB2_OUT_DIR)
```

Comment:
I think `@` can be removed since the command log is not so redundant. 
It is better to remove it so that the operation will be clearer.

Reply:
> I think `@` can be removed since the command log is not so redundant. It is better to remove it so that the operation will be clearer.

I removed `@` in Makefile.